static/src/javascripts/projects/common/views/commercial/creatives/logo-ad-feature.html

```diff
@@ -3,8 +3,4 @@ <h3 class="ad-slot--paid-for-badge__header"><%=logoFeatureLabel%></h3>
     <a href="<%=clickMacro%><%=logoUrl%>" class="ad-slot--paid-for-badge__link">
         <img class="ad-slot--paid-for-badge__logo" src="<%=logoImage%>" />
     </a>
-    <% if (!guardian.config.switches.newCommercialContent) { %>
```

Comment:
I'm pretty sure the code should stay here and the test become
```
<% if (!guardian.config.switches.isAdvertisementFeature) { %>
```

This is because the labels should appear for sponsored content—but it's best to double-check that.